src/MongoDB.EntityFrameworkCore/Serializers/EntitySerializer.cs

```diff
@@ -22,6 +22,16 @@
 
 namespace MongoDB.EntityFrameworkCore.Serializers
 {
+    internal static class EntitySerializer
+    {
+        public static IBsonSerializer Create(IReadOnlyEntityType entityType)
```

Comment:
This helper method makes it easier to instantiate instances of `EntitySerializer<TValue>` when `TValue` is not known until runtime.